src/test/kotlin/pt/up/fe/ni/website/backend/controller/PostControllerTest.kt

```diff
@@ -174,22 +236,6 @@ internal class PostControllerTest @Autowired constructor(
                 @DisplayName("size must be greater or equal to ${Constants.Body.minSize}()")
                 fun size() = validationTester.hasMinSize(Constants.Body.minSize)
             }
-
-            @Nested
-            @DisplayName("thumbnailPath")
-            @TestInstance(TestInstance.Lifecycle.PER_CLASS)
-            inner class ThumbnailValidation {
-                @BeforeAll
-                fun setParam() {
-                    validationTester.param = "thumbnailPath"
-                }
-
-                @Test
-                fun `should be required`() = validationTester.isRequired()
-
-                @Test
-                fun `should not be empty`() = validationTester.isNotEmpty()
-            }
```

Comment:
Why did this get removed?

Reply:
We could also add a SlugValidation class to test the size and null_or_not_blank annotations

Reply:
This didn't get removed, I just moved a test that was after validation to be before.